includes/class-wcpdf-settings-debug.php

```diff
@@ -36,120 +38,331 @@ public function debug_tools( $tab, $section ) {
 
-						printf('<div class="notice notice-success"><p>%s</p></div>', esc_html__( 'Legacy settings deleted!', 'woocommerce-pdf-invoices-packing-slips' ) );
-					}
-					?>
-				</form>
-			</p>
-			<p>
-				<a href="<?php echo esc_url( admin_url( 'admin.php?page=wpo-wcpdf-setup' ) ); ?>" class="button"><?php esc_html_e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
-			</p>
+							printf('<div class="notice notice-success"><p>%s</p></div>', esc_html__( 'Legacy settings deleted!', 'woocommerce-pdf-invoices-packing-slips' ) );
+						}
+						?>
+					</form>
+				</div>
+				<div class="tool">
+					<h4><?php _e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></h4>
+					<p><?php _e( 'If you miss the initial configuration of this plugin when you installed it, you could do it again below.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
+					<a href="<?php echo esc_url( admin_url( 'admin.php?page=wpo-wcpdf-setup' ) ); ?>" class="button"><?php esc_html_e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
+				</div>
+				<div class="tool">
+					<h4><span><?php _e( 'Export Settings', 'woocommerce-pdf-invoices-packing-slips' ); ?></span></h4>
+					<p><?php _e( 'Download the settings of the plugin for this website as a .json file. This allows for effortless importation of the configuration into another site.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
+					<form class="wpo_wcpdf_debug_tools_form" method="post">
+						<input type="hidden" name="debug_tool" value="export-settings">
+						<fieldset>
+							<select name="type" required>
+								<?php
+									foreach ( $this->get_setting_types() as $type => $name ) {
+										?>
+										<option value="<?= $type; ?>"><?php printf( /* translators: type of settings */ __( "%s Settings", 'woocommerce-pdf-invoices-packing-slips' ), $name ); ?></option>
+										<?php
+									}
+								?>
+							</select>
+							<a href="" class="button button-secondary submit"><?php _e( 'Export', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
+						</fieldset>
+						<fieldset>
+							<div class="notice inline" style="display:none;"><p></p></div>
+						</fieldset>
+					</form>
+				</div>
+				<div class="tool">
+					<h4><span><?php _e( 'Import Settings', 'woocommerce-pdf-invoices-packing-slips' ); ?></span></h4>
+					<p><?php _e( 'Bring in the plugin settings from a .json file. This file can be obtained by downloading the settings on another site via the form above.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
+					<form class="wpo_wcpdf_debug_tools_form" method="post" enctype="multipart/form-data">
+						<input type="hidden" name="debug_tool" value="import-settings">
+						<fieldset>
+							<input type="file" name="file" accept="application/json" required>
+							<a href="" class="button button-secondary submit"><?php _e( 'Import', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
```

Comment:
This is only a suggestion: Although the loading time for importing and exporting files is quite short, it would be interesting to add a loader, perhaps to the right of the buttons.

Reply:
I'm using the Block UI library already, but I can improve that.